src/assets/css/blog.scss

```diff
@@ -0,0 +1,97 @@
+$article-l-width: 324px;
+$article-xxl-width: 424px;
+$article-padding: 19px;
+
+$max-featured-image-height: 236px;
+
+$title-color: #291c48;
+
+$date-color: #54556b;
+$date-font-size: 12px;
+
+$excerpt-margin: 13px;
```

Comment:
All these variables are "local" (or specific to that page), which is why they are there and not in the `variables.scss` file.